index.html

```diff
@@ -112,20 +111,6 @@
                 Twitter</a> or <a href="https://github.com/berlinjs/berlinjs.org">create a pull request on GitHub</a> to this page right away.
               </p>
             </div>
-
-            <!--
```

Comment:
This appears to be a duplicate of line 66:
    <!-- THIS IS WHAT A TALK ABSTRACT LOOKS LIKE